test/WebJobs.Extensions.Http.Tests/WebJobs.Extensions.Http.Tests.csproj

```diff
@@ -5,6 +5,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <!-- We use a package reference to avoid building this in CI. -->
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions" Version="5.1.0" />
```

Comment:
`WebJobs.Extensions.Timer.Storage` was updated to `5.0.0` - why is this one `5.1.0`? Latest releases: https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions/#versions-body-tab.

Reply:
Fixed that. Ran into another issue where our internal nuget feeds got a bad version pushed to it.